keel-web/src/main/kotlin/com/netflix/spinnaker/keel/apidocs/GenericWildcardTypeModelConverter.kt

```diff
@@ -0,0 +1,65 @@
+package com.netflix.spinnaker.keel.apidocs
+
+import com.fasterxml.jackson.databind.type.SimpleType
+import io.swagger.v3.core.converter.AnnotatedType
+import io.swagger.v3.core.converter.ModelConverter
+import io.swagger.v3.core.converter.ModelConverterContext
+import io.swagger.v3.oas.models.media.Schema
+import java.lang.reflect.ParameterizedType
+import java.lang.reflect.WildcardType
+import org.springframework.stereotype.Component
+
+/**
+ * Kotlin-ized version of a class provided by [bnasslahsen](https://github.com/bnasslahsen).
```

Comment:
Maybe a pointer to the actual code you used as a reference?

Reply:
It’s literally just a text file he attached to that issue